### DISCUSSION TITLE
get latest reply

### DISCUSSION THREAD

**Discussion by vespino** [php-mime-mail-parser/php-mime-mail-parser#392] [Q&A] [answered]
Is it possible to filter the latest reply from an email?

**Comment by eXorus** [comment, marked as answer]
No it's not possible and not in the scope of this library.
You have specialized library to parse the content of an email to detect the replies, it's very complicated and not very accurate.

**Reply by vespino** [reply]
Thanks. Any tips on which libraries I could use?

**Reply by eXorus** [reply]
https://github.com/search?l=PHP&q=email+reply+parser&type=Repositories

I don't know any.

**Reply by vespino** [reply]
Thanks, I'll have a look.
